docs/time/index.rst

```diff
@@ -174,6 +174,10 @@ yday         :class:`~astropy.time.TimeYearDayTime`             2000:001:00:00:0
    initialization.  Furthermore, any specific realization information,
    such as ``UT(NIST)`` is stored only as long as the time scale is not changed.
 .. [#] `Rots et al. 2015, A&A 574:A36 <http://adsabs.harvard.edu/abs/2015A%26A...574A..36R>`_
+
```

Comment:
In general one should not change code or text that is unrelated to your new content.  So back out this change to add 4 lines here.